### PR TITLE
Remove qpid ports

### PR DESCRIPTION
**Pull request by ehelms**
#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_enabling-connections-from-capsule-to-satellite.adoc
Comment (ekohl) [inline guides/common/modules/proc_enabling-connections-from-capsule-to-satellite.adoc:1]: I think this whole file is no longer needed. Without the port there's no need to make the rules persistent. Then there's no procedure left.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by maximiliankolb** [comment]
> These ports are still in `guides/image-sources/load-balancing-architecture.drawio` but I am not aware of how to update that.

This is a source file for draw.io:

* https://docs.theforeman.org/nightly/Configuring_Load_Balancer/index-katello.html#components-of-a-load-balanced-setup_load-balancing (in Foreman blue; commited to git as PNG)
* https://docs.theforeman.org/nightly/Configuring_Load_Balancer/index-orcharhino.html#components-of-a-load-balanced-setup_load-balancing (in orcharhino green; commited as SVG)
* https://docs.theforeman.org/nightly/Configuring_Load_Balancer/index-satellite.html#components-of-a-load-balanced-setup_load-balancing (PNG for Satellite, but same idea)

I can provide a patch to fix the `.drawio` and `.svg` files; for Satellite, I'll have to ask on Monday who is best to make this change.

@ehelms Just to clarify for Foreman: We need to remove the third arrow for port 5647. ACK?

**Comment by ehelms** [comment]
> @ehelms Just to clarify for Foreman: We need to remove the third arrow for port 5647. ACK?

Yep!

**Review by maximiliankolb — COMMENTED** [review]
diff LGTM; I have added the required changes for graphics to my own TODO.

Related PR for firewalld: https://github.com/firewalld/firewalld/pull/1319#issuecomment-2145190377

**Comment by maximiliankolb** [comment]
I propose to merge this as is with a technical ACK. Afterwards, @bangelic and I can provide follow-up PRs to remove the port from the images.

**Comment by bangelic** [comment]
@ehelms @maximiliankolb A Jira ticket would need to be created to submit to the graphics team to update or remove something from an existing image. This ticket I raised a while back can be used as a template: https://issues.redhat.com/browse/DIAGRAMS-427

**Review by evgeni — APPROVED** [review]
ack on the change, leaving it unmerged until you've sorted the diagrams thing

**Comment by ehelms** [comment]
I think diagrams are sorted over here -- https://github.com/theforeman/foreman-documentation/pull/3268

**Review by maximiliankolb — APPROVED** [review]

**Pull request merged**